packages/salesforcedx-apex-replay-debugger/src/messages/i18n.ts

```diff
@@ -23,5 +23,10 @@ export const messages = {
   no_log_file_text:
     'The log file either is missing or does not have any log lines in it.',
   incorrect_log_levels_text:
-    'The log must be generated with log categories Apex code at the FINEST level and Visualforce at the FINER level.'
+    'The log must be generated with log categories Apex code at the FINEST level and Visualforce at the FINER level.',
+  unable_to_retrieve_org_info: 'Unable to retrieve OrgInfo',
+  heap_dump_error:
+    'Problems were encountered while retrieving heap dump information. Message=%s, ErrorCode=%s. Heap Dump Information=%s',
+  heap_dump_error_wrap_up_text:
```

Comment:
It's worth nothing that I'd worked with Ruth on this error message. She's OOO which is why I didn't add her to the review.

Reply:
Please make such conversations public next time for traceability. It's perfectly fine to have your draft of the text in the PR and iterate with @ruthemmanuelle on it.

Reply:
@DatGuyJonathan we did it quickly over slack late Wednesday afternoon because I noticed her OOO message otherwise we'd have done it over email, as per usual.

Reply:
@JimSuplizio , there's no need to add an extra step by asking me to review the messages via email or Slack. We can just review them on the PR in the future. If you think there's a good reason to review them elsewhere, we can talk about it. Maybe we should put it on our parking lot agenda for standup?